src/Camelotia.Services/Models/FolderModel.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.Collections.Generic;
+
+namespace Camelotia.Services.Models
+{
+    public class FolderModel
+    {
+        public string FullPath { get; set; }
+        public string Name { get; set; }
+        public IEnumerable<string> Children { get; set; }
```

Comment:
I'd suggest adding a constructor accepting these arguments, and making the properties get-only.